Makefile

```diff
@@ -279,10 +282,17 @@ verify-extended: check-generate check format test-cov test-cov-clean test-integr
 
 kind-up kind-down gardener-up gardener-down register-local-env tear-down-local-env register-kind2-env tear-down-kind2-env test-e2e-local-simple test-e2e-local-migration test-e2e-local: export KUBECONFIG = $(GARDENER_LOCAL_KUBECONFIG)
 kind2-up kind2-down gardenlet-kind2-up gardenlet-kind2-down: export KUBECONFIG = $(GARDENER_LOCAL2_KUBECONFIG)
+kind-extensions-up kind-extensions-down gardener-extensions-up gardener-extensions-down: export KUBECONFIG = $(GARDENER_EXTENSIONS_KUBECONFIG)
 kind-ha-single-zone-up kind-ha-single-zone-down gardener-ha-single-zone-up register-kind-ha-single-zone-env tear-down-kind-ha-single-zone-env ci-e2e-kind-ha-single-zone: export KUBECONFIG = $(GARDENER_LOCAL_HA_SINGLE_ZONE_KUBECONFIG)
 kind-ha-multi-zone-up kind-ha-multi-zone-down gardener-ha-multi-zone-up register-kind-ha-multi-zone-env tear-down-kind-ha-multi-zone-env ci-e2e-kind-ha-multi-zone: export KUBECONFIG = $(GARDENER_LOCAL_HA_MULTI_ZONE_KUBECONFIG)
 kind-operator-up kind-operator-down operator-up operator-down test-e2e-local-operator ci-e2e-kind-operator: export KUBECONFIG = $(GARDENER_LOCAL_OPERATOR_KUBECONFIG)
 
+gardener-extensions-up gardener-extensions-down: export SEED_NAME = $(GARDENER_EXTENSIONS_SEED_NAME)
```

Comment:
Does it make sense to already use a concrete seed name, one that has the provider name inside? Background: When we enhance the setup later to support multiple seeds, then the already existing one has the expected name and People don't need to bother with recreating, etc. 🙂

Reply:
I'm not sure what you mean 😅 
There is a concrete seed name (provider-extensions) and you can set a different one with provider name included via `GARDENER_EXTENSIONS_SEED_NAME`

Reply:
What I meant is to name the seed after the provider which is used. As soon as we add support for multiple seeds, a generic name as `provider-extensions` won't work any more and devs have to recreate their seeds. Nothing dramatic but will be a relief for later. Please consider this a proposal from my side, we can also proceed for now if you prefer.

Reply:
As discussed, we'll proceed with this approach for now since more changes will be required for supporting multiple seeds.